web-ui/src/components/guild-results/GuildResults.jsx

```diff
@@ -93,13 +94,17 @@ const GuildResults = () => {
                 onClose={handleClose}
                 onSave={async guild => {
                   if (csrf) {
-                    let res = await createGuild(guild, csrf);
-                    let data =
-                      res.payload && res.payload.data && !res.error
+                    const res = await createGuild(guild, csrf);
+                    const data = res.payload?.data && !res.error
                         ? res.payload.data
                         : null;
                     if (data) {
                       dispatch({ type: ADD_GUILD, payload: data });
+                    const resGuildLeader = await getGuildLeaders(data.id, csrf);
```

Comment:
Looks like the indentation is off on this lines. We have Prettier configured for the project now, so you can configure your IDE to run it.

Reply:
Is there any docs for setting that up. Mine isn't picking up the config in intellij.

Reply:
I only know how to configure it in VS Code.
Maybe this will do it: https://www.jetbrains.com/help/idea/prettier.html

Reply:
Fixed. Using vs code.